spyder/utils/icon_manager.py

```diff
@@ -240,6 +240,7 @@
     'DialogCloseButton':       [('fa.close',), {'color': MAIN_FG_COLOR}],
     'DirClosedIcon':           [('fa.folder-o',), {'color': MAIN_FG_COLOR}],
     'DialogHelpButton':        [('fa.life-ring',), {'color': 'darkred'}],
+    'TutorialVideosButton':    [('fa5s.video',), {'color': MAIN_FG_COLOR}],
```

Comment:
Let's call this `VideoIcon` to make it a bit more generic (in case we want to use in other places later on).